commit ad96495e9b26e46adf7f0b89542dd60aea544dca
Author: Eric Anholt <eric@anholt.net>
Date:   Thu Dec 18 16:07:15 2014 -0800

    mm: Remove the PFN busy warning
    
    See commit dae803e165a11bc88ca8dbc07a11077caf97bbcb -- the warning is
    expected sometimes when using CMA.  However, that commit still spams
    my kernel log with these warnings.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/mm/page_alloc.c b/mm/page_alloc.c
index 34ada718ef47..3fa01a5280db 100644
--- a/mm/page_alloc.c
+++ b/mm/page_alloc.c
@@ -7297,8 +7297,6 @@ int alloc_contig_range(unsigned long start, unsigned long end,
 
 	/* Make sure the range is really isolated. */
 	if (test_pages_isolated(outer_start, end, false)) {
-		pr_info("%s: [%lx, %lx) PFNs busy\n",
-			__func__, outer_start, end);
 		ret = -EBUSY;
 		goto done;
 	}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1004_ad96495e9b26e46adf7f0b89542dd60aea544dca.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1004_ad96495e9b26e46adf7f0b89542dd60aea544dca.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
